docs/guides/contribute.md

```diff
@@ -33,7 +38,7 @@ Change your working directory to the cloned repository.
 cd plone-sphinx-theme
 ```
 
-Then create a Python virtual environment, install requirements for development, and install the package in development mode with a single command.
+Then install a supported Python version for development, create a Python virtual environment, install requirements for development, and install the package in development mode with a single command.
 
 ```shell
 make dev
```

Comment:
It would be nice, if we have a `uv` command here too!

Reply:
That is what `make dev` does.

https://github.com/plone/plone-sphinx-theme/pull/67/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R31-R34

See relevant docs:

https://docs.astral.sh/uv/guides/install-python/#installing-a-specific-version

Reply:
True story, I had been an end user of GNU Make for decades, and never wrote a line of it in my life, until about three years ago. I found this resource incredibly helpful to understand and use the power of Make. I'm still learning new tricks.

https://makefiletutorial.com/

Reply:
> That is what make dev does.
> 
> https://github.com/plone/plone-sphinx-theme/pull/67/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R31-R34

ah, I somehow overlooked that!  
marking this convo as resolved! thanks @stevepiercy